src/server/verification/verificationAPI.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import passport from "passport"
+import { get } from 'lodash'
+import {type UserRecord, StorageAPI, VerificationAPI } from '../../imports/types'
+import AdminWallet from '../blockchain/AdminWallet'
+
+function wrapAsync(fn) {
```

Comment:
Shouldn't this be imported from 'import { wrapAsync } from 'server-middlewares'?

Reply:
Yes